rubygem-foreman_cockpit/rubygem-foreman_cockpit.spec

```diff
@@ -91,6 +91,9 @@ cp -a .%{gem_dir}/* \
 exit 0
 
 %changelog
+* Sun Apr 10 2016 Daniel Lobato Garcia <elobatocs@gmail.com> 2.0.0-1.fm1_10
```

Comment:
Please leave this out, it won't be for fm1_10 and the changelog will be generated by tito.